inst/extdata/gcam-usa/EIA_elect_td_ownuse_prices.csv

```diff
@@ -0,0 +1,58 @@
+# File: EIA_elect_td_ownuse_prices.csv
+# Title: EIA State Electricity Profiles 2008
+# Description: Table A1. Selected Electric Industry Summary Statistics by State
+# Source: ftp://eia.gov/electricity/stateprofiles/08st_profiles/table_a.pdf
+# Units: In column names
```

Comment:
What is `c.kWh` is Price column?

Reply:
I believe it's cents/kWh (maybe in 2008$)